### PR TITLE
punctuation and wordy

### PR DESCRIPTION
**Pull request by Amalearner**
punctuation and wordy words correction

### CODE REVIEW

File: documentation/asciidoc/computers/os/updating.adoc [hunk 1]
```diff
@@ -26,11 +26,11 @@ Next, *upgrade* all your installed packages to their latest versions with the fo
 sudo apt full-upgrade
 ----
 
-Note that `full-upgrade` is used in preference to a simple `upgrade`, as it also picks up any dependency changes that may have been made.
+Note that `a full-upgrade` is preferred to a simple `upgrade`, as it also picks up any dependency changes that may have been made.
```
Comment (aallan) [inline documentation/asciidoc/computers/os/updating.adoc:29]: Incorrect the `full-upgrade` and `upgrade` are command line flags.

File: documentation/asciidoc/computers/os/updating.adoc [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 
-It's important to keep your Raspberry Pi up to date. The first and probably the most important reason is security. A device running Raspberry Pi OS contains millions of lines of code that you rely on. Over time, these millions of lines of code will expose well-known vulnerabilities, which are documented in https://cve.mitre.org/index.html[publicly available databases] meaning that they are easy to exploit. The only way to mitigate these exploits as a user of Raspberry Pi OS is to keep your software up to date, as the upstream repositories track CVEs closely and try to mitigate them quickly.
+It's important to keep your Raspberry Pi up to date. The first and probably the most important reason is security. A device running Raspberry Pi OS contains millions of lines of code you rely on. Over time, these millions of lines of code will expose well-known vulnerabilities, which are documented in https://cve.mitre.org/index.html[publicly available databases], meaning they are easy to exploit. The only way to mitigate these exploits as a user of Raspberry Pi OS is to keep your software up to date, as the upstream repositories track CVEs closely and try to mitigate them quickly.
```
Comment (aallan) [inline documentation/asciidoc/computers/os/updating.adoc:3]: Disagree with the removal of words here.

File: documentation/asciidoc/computers/os/updating.adoc [hunk 3]
```diff
@@ -26,11 +26,11 @@ Next, *upgrade* all your installed packages to their latest versions with the fo
...
+Note that `a full-upgrade` is preferred to a simple `upgrade`, as it also picks up any dependency changes that may have been made.
 
-Generally speaking, doing this regularly will keep your installation up to date for the particular major Raspberry Pi OS release you are using (e.g. Buster). It will not update from one major release to another, for example, Stretch to Buster or Buster to Bullseye.
+Generally speaking, doing this regularly will keep your installation up to date for the particular major Raspberry Pi OS release you are using (e.g., Buster). It will not update from one major release to another, for example, Stretch to Buster or Buster to Bullseye.
 
-However, there are occasional changes made in the Raspberry Pi OS image that require manual intervention, for example a newly introduced package. These are not installed with an upgrade, as this command only updates the packages you already have installed.
+However, there are occasional changes made in the Raspberry Pi OS image that require manual intervention, such as a newly introduced package. These are not installed with an upgrade, as this command only updates the packages you already have installed.
```
Comment (aallan) [inline documentation/asciidoc/computers/os/updating.adoc:33]: Pointless change?

Comment (Amalearner) [reply documentation/asciidoc/computers/os/updating.adoc:33]: I'm reading it as a writer. But, it's fine. Still on other changes.

**Pull request closed without merging**